volosheniukym/second/task.1.py

```diff
@@ -0,0 +1,36 @@
+'''
+Використовуючи регулярні вирази, для поданого нижче тексту замініть кожне входження дати у форматі
+ `31.12.2019` на дату у форматі `Dec 31, 2019`. Виведіть результат.
+'''
+
+
+import re
+
+string = input("Введіть текст з датою\n")
+
```

Comment:
You can give an example what format do you need (different regions using different format for data).